tools/releases/dockerfiles/Dockerfile.kuma-init

```diff
@@ -17,9 +15,6 @@ COPY /tools/releases/templates/LICENSE \
 
 COPY /tools/releases/templates/NOTICE-kumactl /kuma/NOTICE
 
-# ebpf
-COPY --from=ebpf /ebpf /kuma/ebpf
```

Comment:
Only have one question. How do we get ebpf files copied to the image now?

Reply:
they are compiled into `kumactl` binary and then when installing tproxy, they are taken from the binary and saved to the filesystem